salt/addons/registries-operator/init.sls

```diff
@@ -0,0 +1,28 @@
+include:
+  - docker
+  - kubectl-config
+  - kube-apiserver
+
+{% from '_macros/kubectl.jinja' import kubectl, kubectl_apply_dir_template with context %}
+
+/etc/kubernetes/addons/registries-operator/registries-operator.yaml:
+  file.managed:
+    - source:   "salt://addons/registries-operator/manifests/registries-operator.yaml"
+    - user:     root
+    - group:    root
+    - mode:     0600
+    - template: jinja
+    - makedirs: true
+
+{{ kubectl_apply_dir_template("salt://addons/registries-operator/manifests/",
+                              "/etc/kubernetes/addons/registries-operator/") }}
+
+{% set certs = salt.caasp_docker.get_registries_certs(salt.caasp_pillar.get('registries', [])) %}
+
+apply registries operator resources:
```

Comment:
Even when they are valid names in Salt, I think we jhave never used this kind of ids for Salt: we have always followed the `thihs-is-an-id` convention.